Pak-Je-Kans.md

```diff
@@ -0,0 +1,48 @@
+# Inleiding
+---
+In dit bestand ga ik uitleggen wat er gemaakt moet worden. Ook ga ik beschrijven wat dit betekend voor de back end. Ik zal ook de benodigdheden zoals informatie en tools die handig zijn noemen en beschrijven. Zo kan ik het PIO project beter inzien.
```

Comment:
Persoonlijke Inkomens Omgeving (PIO)
Wat is PIO? Toevallig weet ik het nu, maar over een jaar misschien niet meer. 1 keer de afkorting voluit schrijven kan daarbij helpen bij de absolute wildgroei aan drie letterige afkortingen (DLA) bij de gemeente.